config/database.js

```diff
@@ -89,17 +89,17 @@ const getBook = (id) => {
   WHERE books.id = ${id}`)
 }
 
-const getBooks = ({ page, title, author, year, cont}) => {
-  page = parsInt( page || 1 )
+const getBooks = ({ page, title, author, year, count}) => {
```

Comment:
noted and thanks dev,  I will make sure to ccreate a script that builds the database and injects the schema.sql => all the tables into our database.  Also i will take into consideration how to improve my naming conventions for my fellow teammates. 🥇